### ISSUE TITLE
3.0.1:  excludes in build.gradle not working

### ISSUE THREAD

**Issue by mhuebner** [bertramdev/asset-pipeline#256]
When I configure excludes in build.gradle these files are also bundled into the war file.

Config example:
```
assets {
    minifyJs = true
    minifyCss = true
    excludes = ['.sass-cache/**/*','sass/**/*']
}
```
But these files are included in the resulting war-package.

**Comment by davydotcom1** [comment]
where is the .sass-cache folder and sass folder in your assets structure

**Comment by mhuebner** [comment]
They live in grails-app/assets/.sass-cache

**Comment by davydotcom** [comment]
hmm we need to probably omit hidden directories from filesystem asset resolver

**Comment by davydotcom** [comment]
The excludes is relative to one more level down not the top `assets` folder.

**Comment by mhuebner** [comment]
Could you tell me which directory this would be? Do you mean relative to `grails-app`?

**Comment by davydotcom1** [comment]
No excludes is relative to the first level under assets I.e. assets/javascripts or assets/libs

**Comment by davydotcom1** [comment]
Really we should never be bundling in hidden directories at all like this . I'll make a nod to exclude .sass-cache from the file system resolver and have an update out today

**Comment by mhuebner** [comment]
I see. I wanted to exclude some other directories like ´node_modules´. All these directories live under ´assets/´. So I would have to exclude ´../sass/**/*´ for example?

**Comment by davydotcom1** [comment]
Sometimes what I do is make a nested assets/libs folder then stick things like node_modules in there so I can exclude

**Comment by mhuebner** [comment]
Would it make sense to move the exclude-Path to `assets`directory level? I think it could make the whole thing a lot more easy to use.

**Comment by davydotcom** [comment]
Not really feasible since the scanner uses the layered file system resolver as a scan path. This could be a jar file or a class path resolver.

**Comment by davydotcom** [comment]
should be fixed in 2.3.7 of ap-core
